pydantic/fields.py

```diff
@@ -340,13 +340,35 @@ def __repr_args__(self) -> ReprArgs:
                     yield s, value
 
 
-def Field(
+class AliasPath:
+    def __init__(self, first_arg: str, *args: str | int) -> None:
+        self.paths = [first_arg] + list(args)
```

Comment:
Minor point, but I would change this attribute's name from `paths` to `path` since the path is the sequence of fields, not each field on its own. I think it would be less of an issue if the name of the class as a whole didn't already have `Path` in the name.

Reply:
renamed to `path`